Description: Updates calling of the perl interpreter to what we expect in Debian
 More precisely: Call /usr/bin/perl directly instead of using some
 shell magic to locate perl and run it.
 .
 NOTE: only some script use the "-w" or even the "-T" flag for perl.
 This should be the default actually.
Author: Sven Mueller <debian@incase.de>
Forwarded: no
Reviewed-By: Xavier Guimard <yadd@debian.org
Last-Update: 2020-02-10

--- a/perl/imap/cyradm.sh
+++ b/perl/imap/cyradm.sh
@@ -1,6 +1,4 @@
-#! /bin/sh
-exec perl -x -S $0 ${1+"$@"} # -*-perl-*-
-#!perl
+#!/usr/bin/perl
 #
 # Copyright (c) 1994-2008 Carnegie Mellon University.  All rights reserved.
 #
--- a/perl/sieve/scripts/installsieve.pl
+++ b/perl/sieve/scripts/installsieve.pl
@@ -1,6 +1,4 @@
-#! /bin/sh
-exec perl -x -S $0 ${1+"$@"} # -*-perl-*-
-#!perl -w
+#!/usr/bin/perl
 #
 # Copyright (c) 1994-2008 Carnegie Mellon University.  All rights reserved.
 #
--- a/tools/rehash
+++ b/tools/rehash
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # Copyright (c) 1994-2012 Carnegie Mellon University.  All rights reserved.
 #
--- a/tools/translatesieve
+++ b/tools/translatesieve
@@ -1,6 +1,4 @@
-#!/bin/sh
-exec perl -x -S $0 ${1+"$@"} # -*-perl-*-
-#!perl -w
+#!/usr/bin/perl
 #
 # script to translate sieve scripts to use unixhierarchysep and/or altnamespace
 # make sure you run it as the cyrus user
